nestkernel/event_delivery_manager.cpp

```diff
@@ -726,7 +720,6 @@ EventDeliveryManager::gather_target_data( const thread tid )
 
     const bool distribute_completed = distribute_target_data_buffers_( tid );
     gather_completed_checker_[ tid ].logical_and( distribute_completed );
-#pragma omp barrier
 
```

Comment:
Agree here: Superfluous barrier because of following invocation of gather_completed_checker_.any_false().